src/com/xilinx/rapidwright/design/blocks/PBlockGenerator.java

```diff
@@ -68,7 +68,7 @@
 	
 	/** X dimension over Y dimension */
 	public float ASPECT_RATIO = 0.125f;//1.5f;	
-	public float OVERHEAD_RATIO = 1.5f;//1.25f;
+	public float OVERHEAD_RATIO = 4.5f;//1.25f;
```

Comment:
@ElaHobby  - I think we probably want to keep the OVERHEAD_RATIO default to 1.5.  I'm sure you have found instances where it has needed to be increased, but we probably aren't ready to change it for everyone.